index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+    <head>
+      <link rel="stylesheet" href="style.css">
+
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+    <body>
+        <header>
+        <section class="toolbar">
+            <div class="tbar">
+                <div class="menu-bar">
+                    <p class="john-doe">IKOOTE</p>
+          
+            <img src="./images/Icon - Menu.png" class="icon-Menu">
+                </div>
+            </div>
+
+        </section>
+        <section class="headline">
+            <div> 
+                <h1>Hey There. I’m Ikoote </h1>
+            <h2>I’am a Software Developer</h2>
+            <p class="txt">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+            </div>
+            <p style="background-image: url('./images/Illustration.svg');" class="illust"> 
+           
+            <div>
+                <ul>
+                    <li><img src="./images/Vector1.png" alt="github"></li>
+                    <li><img src="./images/Vector2.png" alt="instagram"></li>
+                    <li><img src="./images/Vector3.png" alt="tree"></li>
+                    <li><img src="./images/Vector4.png" alt="twitter"></li>
+                    <li><img src="./images/Vector5.png" alt="messenger"></li>
+                </ul>
```

Comment:
- [ ] Great work making the changes requested by the previous reviewer. However, you forgot to wrap each image in the `a` tag. This is the idea, when I click each icon, it should direct me to the social media page of that link. Kindly wrap each image in the `a` tag. 👍 

#### Hint:

```HTML
 <ul>
   <li><a href="..."><img src="./images/Vector1.png" alt="github"></a></li>
    ...
 </ul>
```